src/main/java/uk/gov/hmcts/darts/common/repository/ExternalObjectDirectoryRepository.java

```diff
@@ -65,6 +65,12 @@ List<ExternalObjectDirectoryEntity> findByMediaStatusAndType(MediaEntity media,
     List<ExternalObjectDirectoryEntity> findByTranscriptionDocumentEntityAndExternalLocationType(TranscriptionDocumentEntity transcriptionDocument,
                                                                                                  ExternalLocationTypeEntity externalLocationType);
 
+
+    List<ExternalObjectDirectoryEntity> findByTranscriptionDocumentEntityAndExternalLocationTypeAndStatus(
```

Comment:
I forgot to mention that you need to add integration tests for all new repository methods. I think its in the best practises page.

Reply:
This is spring native query so it would have been tested apart of the spring framework, but for custom queries I would agree

Reply:
So you will need to add repository integration tests for your changes to ObjectAdminActionRepository

Reply:
I must have missed that but yes agreed will add now :)

Reply:
There were existing tests that cover this but i have added additional ones to cover isDeleted = true scenarios + where we only have media or transcription documents not both